data/hydraulic-data.py

```diff
@@ -0,0 +1,147 @@
+"""Create hydraulic systems dataset"""
+
+import os
+import zipfile
+# call on members Union, List
+import typing
+# call on members data.CustomData, systemutils_more.download, systemutils.config
+import h2oaicore
+import datatable as dt
+import numpy as np
+import pandas as pd
```

Comment:
Just a style comment, very much only a bit:
```
Imports should be grouped in the following order:

    Standard library imports.
    Related third party imports.
    Local application/library specific imports.

You should put a blank line between each group of imports.
```

Reference: https://www.python.org/dev/peps/pep-0008/#imports


In order words:
```
import os
import typing
import zipfile

import datatable as dt
import numpy as np
import pandas as pd

import h2oaicore
```

Also in alphabetical order and removed comments unless necessary.